services/src/main/java/org/keycloak/authentication/authenticators/x509/AbstractX509ClientCertificateAuthenticatorFactory.java

```diff
@@ -138,14 +139,18 @@
         crlDPEnabled.setName(ENABLE_CRLDP);
         crlDPEnabled.setDefaultValue(false);
         crlDPEnabled.setLabel("Enable CRL Distribution Point to check certificate revocation status");
-        crlDPEnabled.setHelpText("CRL Distribution Point is a starting point for CRL. CDP is optional, but most PKI authorities include CDP in their certificates.");
+        crlDPEnabled.setHelpText("CRL Distribution Point is a starting point for CRL. If this is ON, then CRL checking will be done based on the CRL distribution points included" +
+                " in the checked certificates. CDP is optional, but most PKI authorities include CDP in their certificates.");
 
         ProviderConfigProperty cRLRelativePath = new ProviderConfigProperty();
-        cRLRelativePath.setType(STRING_TYPE);
+        cRLRelativePath.setType(MULTIVALUED_STRING_TYPE);
         cRLRelativePath.setName(CRL_RELATIVE_PATH);
         cRLRelativePath.setDefaultValue("crl.pem");
-        cRLRelativePath.setLabel("CRL File path");
-        cRLRelativePath.setHelpText("The path to a CRL file that contains a list of revoked certificates. Paths are assumed to be relative to $jboss.server.config.dir");
+        cRLRelativePath.setLabel("CRL Path");
+        cRLRelativePath.setHelpText("Applied just if CRL checking is ON and CRL Distribution point is OFF. It contains the URL (typically 'http' or 'ldap') " +
+                " where the CRL is available. Alternatively it can contain the path to a CRL file that contains a list of revoked certificates. Paths are assumed to be relative to $jboss.server.config.dir ." +
```

Comment:
Together with end of previous row, two spaces; at the end there should be dot and space swapped.
```suggestion
                "where the CRL is available. Alternatively it can contain the path to a CRL file that contains a list of revoked certificates. Paths are assumed to be relative to $jboss.server.config.dir. " +
```

Reply:
Fixed, Thanks